curvature/collector.py

```diff
@@ -171,6 +172,7 @@ def ways_callback(self, ways):
 							self.routes[way['name']] = []
 						self.routes[way['name']].append(way)
 					else:
+						way['constituents'] = [copy.copy(way)]
```

Comment:
This is more a hack, than a proper solution to the problem, solving it in a cleaner way would require refactoring a larger portion of the code base.